client/client.go

```diff
@@ -36,6 +37,7 @@ func Get(ctx context.Context, url string, tracer trace.Tracer) error {
 }
 
 type tracerProvider struct {
```

Comment:
I think we can get rid of this type. This `Get` functions should be accepting `TracerProvider` as argument, not `Tracer`, it will be more aligned with how OTEL SDK is meant to be used.